lightning_examples/reinforce-learning-DQN/dqn.py

```diff
@@ -363,7 +367,7 @@ def get_device(self, batch) -> str:
 
 trainer = Trainer(
     gpus=AVAIL_GPUS,
-    max_epochs=200,
+    max_epochs=400,
```

Comment:
```suggestion
    max_epochs=200,
```
is this needed? there is some timeout issue if this takes too long...

Reply:
Is this why the code initially failed the automated tests? Specifically, it failed code block 8. This is the block that calls trainer.fit(model). I couldn't figure out how to see why it was failing the test, so I wasn't sure what to change.

In general, how do I tell what the error is when I push code that fails one of the automatic tests?